code/modules/antagonists/changeling/powers/absorb.dm

```diff
@@ -41,7 +42,7 @@
 				to_chat(target, "<span class='danger'>You feel a sharp stabbing pain!</span>")
 				target.take_overall_damage(40)
 		SSblackbox.record_feedback("nested tally", "changeling_powers", 1, list("Absorb DNA", "[stage]"))
-		if(!do_mob(user, target, 150))
+		if(!do_mob(user, target, 5 SECONDS))
```

Comment:
PR description says absorbtion is decreased from 45 seconds to 15 seconds but this only reduces it by 10 seconds, is this correct?

Reply:
yeah it's ran three times